platforms/web/public/locales/en/account.json

```diff
@@ -22,14 +22,17 @@
     "discount_period_other": "For the first {{count}} {{period}}",
     "free_trial_discount": "Free trial",
     "go_back_to_checkout": "Go back to checkout",
+    "redirecting_to_paypal": "Redirecting to PayPal",
     "invalid_card_expiry": "Invalid card expiry",
     "invalid_card_number": "Invalid card number",
     "invalid_cvc_number": "Invalid CVC number",
+    "card_details_updated": "Your card details have been updated",
     "monthly": "Monthly subscription",
     "no_payment_needed": "No payment needed",
     "payment_cancelled": "Payment cancelled",
     "payment_cancelled_message": "You have cancelled the payment",
     "payment_error": "Payment failed",
+    "payment_success": "Payment was successful",
```

Comment:
We might not always know if the payment was indeed successfully handled from the backend. Maybe it's safer to change this text to something like "Payment was successfully sent"? @Videodock/jw-ott-front-end

Reply:
I've integrated the announcer in specific scenarios where we have confirmation of successful payment processing. For instance, it's triggered right before redirecting to the welcome page once user access is validated, or after we receive confirmation that payment details have been successfully updated by our services. However, considering your point about potential backend uncertainties, I agree that adjusting the message to 'Payment was successfully sent' would be a more accurate representation of the process's state at that point.